src/lib/task.js

```diff
@@ -1012,7 +1012,7 @@ export class Task extends EventEmitter {
 
   @return New queue.
   */
-  createQueue(credentials, runtime) {
+  createQueue(credentials) {
```

Comment:
It doesn't need it now, I removed runtime from being passed in because it was not needed.  Maybe at some point in the past it was needed but that has since changed.

Side note: I looked into why we need/want this method at all, and it seems it's used for injecting a queue client in tests by overriding this method so probably simplest just keeping it here.